course_gather/serializers.py

```diff
@@ -55,3 +26,13 @@ class StateSerializer(serializers.ModelSerializer):
     class Meta:
         model = State
         fields = '__all__'
+
+
+class CourseSerializer(serializers.ModelSerializer):
+    mtu_equiv_id = MTUCourseSerializer(read_only=True)
```

Comment:
it's not really an `id` if it's a full object

Reply:
Fair point.